python-threatexchange/threatexchange/exchanges/impl/fb_threatexchange_api.py

```diff
@@ -124,6 +124,15 @@ def from_threatexchange_json(
 
         for td_json in te_json.raw_json["descriptors"]["data"]:
             td_id = int(td_json["id"])
+            if "owner" not in td_json:
+                # Regrettable bug in threatexchange where descriptor from
+                # `descriptors` is visible but `owner` field is not.
+                # It's on Meta to fix this bug, for now, just drop record.
+                # Dropping record isn't create, because it breaks the correctness
+                # of threat_updates, since it can oscillate between visible and
+                # not separately from the threat_update order.
+                # TODO: Should we log this?
```

Comment:
yes, it would be good to set the debug log level and then check how many of the record are in this state.